plugins/org.jkiss.dbeaver.ext.cubrid/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,15 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: DBeaver Cubrid Support
+Bundle-SymbolicName: org.jkiss.dbeaver.ext.cubrid;singleton:=true
+Bundle-Version: 1.0.0.qualifier
+Bundle-Release-Date: 20230123
+Require-Bundle: org.eclipse.core.runtime,
+ org.jkiss.dbeaver.model;visibility:=reexport,
+ org.jkiss.dbeaver.model.sql;visibility:=reexport,
+ com.google.guava;resolution:=optional,

```

Comment:
I don't see any direct guava usage in the classes. Is this dependency required?

Reply:
I have fixed it.